### PR TITLE
chore(i18n,docs): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

File: docs/i18n/portuguese/how-to-open-a-pull-request.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Alguns exemplos de bons títulos de PRs seriam:
 
 1. Uma vez que as edições tenham sido realizadas, será solicitado que você crie um pull request na página do GitHub do seu fork.
 
-   ![Image - Compare & pull request prompt on GitHub](https://contribute.freecodecamp.org/images/github/compare-pull-request-prompt.png)
+   ![Imagem - Comparar e solicitar o pull request no GitHub](https://contribute.freecodecamp.org/images/github/compare-pull-request-prompt.png)
```
Comment (Sembauke) [inline docs/i18n/portuguese/how-to-open-a-pull-request.md:47]: should "Image" be translated?

Comment (sidemt) [reply docs/i18n/portuguese/how-to-open-a-pull-request.md:47]: @Sembauke I think it's OK to translate. This is used as the alt text for the image here https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=proposing-a-pull-request

File: docs/i18n/japanese/devops.md [hunk 2]
```diff
@@ -863,100 +863,100 @@ pm2 save
 pm2 logs
 ```
 
-## Installing and Updating Azure Pipeline Agents
+## Azure Pipeline エージェントのインストールと更新
 
-See: https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/v2-linux?view=azure-devops and follow the instructions to stop, remove and reinstall agents. Broadly you can follow the steps listed here.
+https://docs.microsoft.com/ja-JP/azure/devops/pipelines/agents/v2-linux?view=azure-devops を参照し、手順に従ってエージェントを停止、削除、再インストールします。 一般的には、ここに記載されている手順に従います。
```
Comment (sidemt) [inline docs/i18n/japanese/devops.md:868]: I changed this URL back to en-us on Crowdin. As far as we remember, we keep URLs to external websites to point to English page.

**Review by sidemt — APPROVED** [review]
LGTM

File: docs/i18n/portuguese/how-to-open-a-pull-request.md (hunk 1 above)
Comment (nhcarrigan) [reply docs/i18n/portuguese/how-to-open-a-pull-request.md:47]: Yep this should be fine

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**